### ISSUE TITLE
3 Tests fail on 15.8.1

### ISSUE THREAD

**Issue by rffontenelle** [ddnet/ddnet#4631]
Building DDNet 15.8.1 on Arch Linux (AUR [ddnet](https://aur.archlinux.org/packages/ddnet/) package), `ninja run_tests` fails on 3 of 4 tests from Sql/MapInfo. Below you find the output and some logs (including versions of dependencies).

Output: 

```
...
[----------] 4 tests from Sql/MapInfo
[ RUN      ] Sql/MapInfo.ExactNoFinish/SQLite
/build/ddnet/src/DDNet-15.8.1/src/test/score.cpp:114: Failure
Expected equality of these values:
  pPlayerResult->m_Data.m_aaMessages[i]
    Which is: "\"Kobra 3\" by Zerodin on Novice, \xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85, 5 points, released 3 hours ago, 0 finishes by 0 tees"
    As Text: ""Kobra 3" by Zerodin on Novice, ★★★★★, 5 points, released 3 hours ago, 0 finishes by 0 tees"
  pLine
    Which is: "\"Kobra 3\" by Zerodin on Novice, \xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85, 5 points, released moments ago, 0 finishes by 0 tees"
    As Text: ""Kobra 3" by Zerodin on Novice, ★★★★★, 5 points, released moments ago, 0 finishes by 0 tees"
[  FAILED  ] Sql/MapInfo.ExactNoFinish/SQLite, where GetParam() = 0x56434c00ef40 (0 ms)
[ RUN      ] Sql/MapInfo.ExactFinish/SQLite
/build/ddnet/src/DDNet-15.8.1/src/test/score.cpp:114: Failure
Expected equality of these values:
  pPlayerResult->m_Data.m_aaMessages[i]
    Which is: "\"Kobra 3\" by Zerodin on Novice, \xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85, 5 points, released 3 hours ago, 1 finish by 1 tee in 01:40 median"
    As Text: ""Kobra 3" by Zerodin on Novice, ★★★★★, 5 points, released 3 hours ago, 1 finish by 1 tee in 01:40 median"
  pLine
    Which is: "\"Kobra 3\" by Zerodin on Novice, \xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85, 5 points, released moments ago, 1 finish by 1 tee in 01:40 median"
    As Text: ""Kobra 3" by Zerodin on Novice, ★★★★★, 5 points, released moments ago, 1 finish by 1 tee in 01:40 median"
[  FAILED  ] Sql/MapInfo.ExactFinish/SQLite, where GetParam() = 0x56434c00ef40 (0 ms)
[ RUN      ] Sql/MapInfo.Fuzzy/SQLite
/build/ddnet/src/DDNet-15.8.1/src/test/score.cpp:114: Failure
Expected equality of these values:
  pPlayerResult->m_Data.m_aaMessages[i]
    Which is: "\"Kobra 3\" by Zerodin on Novice, \xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85, 5 points, released 3 hours ago, 1 finish by 1 tee in 01:40 median"
    As Text: ""Kobra 3" by Zerodin on Novice, ★★★★★, 5 points, released 3 hours ago, 1 finish by 1 tee in 01:40 median"
  pLine
    Which is: "\"Kobra 3\" by Zerodin on Novice, \xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85\xE2\x98\x85, 5 points, released moments ago, 1 finish by 1 tee in 01:40 median"
    As Text: ""Kobra 3" by Zerodin on Novice, ★★★★★, 5 points, released moments ago, 1 finish by 1 tee in 01:40 median"
[  FAILED  ] Sql/MapInfo.Fuzzy/SQLite, where GetParam() = 0x56434c00ef40 (0 ms)
[ RUN      ] Sql/MapInfo.DoesntExit/SQLite
[       OK ] Sql/MapInfo.DoesntExit/SQLite (0 ms)
[----------] 4 tests from Sql/MapInfo (3 ms total)
...
```

Log files:
- [full-log.txt](https://github.com/ddnet/ddnet/files/7928417/full-log.txt)
- [build-log.txt](https://github.com/ddnet/ddnet/files/7928420/build-log.txt)
- [check-log.txt](https://github.com/ddnet/ddnet/files/7928422/check-log.txt)

**Comment by heinrich5991** [comment]
Seems to be a duplicate of #4564.

**Comment by rffontenelle** [comment]
@edg-l Is this suppose to be solved already? I see you disabled and then re-enable the tests in the AUR package, still I'm having issues to build in a clean chroot.

**Comment by def-** [comment]
Not a duplicate I believe. This is a new bug. Somehow the time is considered wrongly on your system. We enter a rank with current time, then it is considered 3 hours ago.

**Comment by def-** [comment]
I can't reproduce it. What does `date` print for you?

**Comment by rffontenelle** [comment]
3 hours is my timezone (UTC-3). 🤔
I'll run `date` tomorrow and will let you know.

**Comment by def-** [comment]
Ok, this is probably timezone dependent then. I guess I never noticed because I'm in UTC+1/2. Let me try to fix.

**Comment by def-** [comment]
Fixed by https://github.com/ddnet/ddnet/pull/4632, but there was already an alternative fix in c76f0f5899

**Comment by rffontenelle** [comment]
AUR package updated with both patches. Thanks!
